content/blog/announce-tonic-0-1.md

```diff
@@ -0,0 +1,129 @@
+
+With async/await’s forthcoming stabilization, we created Tonic to support the new syntax natively. This means clients will support async/await out of the box and server
+implementations can be defined via [`async_trait`]s. This provides an unparalleled experience for
+writing async services quickly and efficiently. Not only does Tonic provide a fully featured gRPC implementation but it also comes with a fully featured, batteries-included, HTTP/2 client and server built
+around [`hyper`], [`tokio`]and [`tower`]. Both the client and server implementation provide TLS backed by
+either [`openssl`] or [`rustls`]. The client provides load balancing, interceptors, timeouts, rate limiting,
+concurrency control and more!
+
+Tonic also boasts strong interoperability and correctness. Every commit is checked that it passes
+the [gRPC interop test cases][interop-cases] against the [`grpc-go`] implementation. Hopefully, more
+languages can be added in the future.
+
+### Features
+
+Out of the gate, the goal for Tonic has been to provide a good batteries included experience. It
+already supports many features, with many more planned! Here is a list of features:
+
+- Implemented in pure Rust (minus [`openssl`] which is optional)
+- Interoperability tested via [`tonic-interop`]
+- Bi-directional streaming
+- Custom metadata
+- Trailing metadata
+- Codegen via [`prost`]
+- Exposes [`tracing`] diagnostics (more to come)
+- Fully featured HTTP/2 client and server
+- TLS backed by either [`openssl`] or [`rustls`]
+- Load balancing powered by [`tower`]
+- Reliability features such as timeouts, rate limiting, concurrency control, and more
+- gRPC interceptors
+- And much more to come
+
+### Overview
+
+_helloworld **client** example:_
+
+{{< highlight rust >}}
+let mut client = GreeterClient::connect("http://[::1]:50051")?;
+let request = Request::new(HelloRequest {
+    name: "hello".into(),
+});
+let response = client.say_hello(request).await?;
+println!("RESPONSE={:?}", response);
+{{< /highlight >}}
+
+This shows how easy it is to build a simple gRPC client using code generated by [`tonic-build`]. Internally,
+`connect` will create the most basic "channel/client" but it is possible to configure this to support
+tls, load balancing, timeouts and more.
+
+_helloworld **server** example:_
+
+{{< highlight rust >}}
+#[tonic::async_trait]
+impl Greeter for MyGreeter {
+    async fn say_hello(&self, req: Request<HelloRequest>)
+        -> Result<Response<HelloReply>, Status>
+    {
+        println!("Got a request: {:?}", req);
+        let reply = HelloReply {
+            message: "Zomg, it works!".into(),
+        };
+        Ok(Response::new(reply))
+    }
+}
+{{< /highlight >}}
+
+Servers are just as easy to implement! Using [`dtolnay`]'s amazing [`async_trait`] crate
+to allow server implementations to use `async fn` in a trait. More examples including ones with TLS, authentication, and streaming are included in the [`tonic-examples`] crate.
```

Comment:
"The `tonic-examples` crate demonstrates more advanced cases with TLS, authentication, streaming, and more." ?